invenio_rdm_migrator/transform/base.py

```diff
@@ -34,7 +35,7 @@ def run(self, entries):
                 continue
 
 
-class Entry(ABC):
+class Entry(ABC, DatetimeMixin):
```

Comment:
Added to the entry not the whole transform, since those usually contain more than one `Entry`

Reply:
But isn't it that the `Entry` is already used for the snapshot pipeline, and assumes strings? I know we share the code of the `Entry` classes between the snapshot and actions pipelines, so I would normalize what the expected input is (which in our case differs only in the `actions` trasnforms.

Reply:
Yes, you are correct. The code is shared, see comment here https://github.com/zenodo/zenodo-rdm/pull/450/files#r1289982130

This was a cut corner, mainly because it would mean lots of duplication to have one for actions and one for snapshot. The coupling happens at `ZenodoRecordTransform`. There was an idea to make the entry classes customizable (in line with https://github.com/inveniosoftware/invenio-rdm-migrator/issues/36). But I think at this point its better to shelve it. WDYT?

Reply:
Discussed in Discord https://discord.com/channels/692989811736182844/1118079987891843093/1139140395285098587